numpy/core/src/multiarray/descriptor.c

```diff
@@ -252,9 +252,16 @@ _convert_from_tuple(PyObject *obj)
     if (PyTuple_GET_SIZE(obj) != 2) {
         return NULL;
     }
-    if (!PyArray_DescrConverter(PyTuple_GET_ITEM(obj,0), &type)) {
-        return NULL;
-    }
+	if (align) {
```

Comment:
all the tab characters here should be four spaces. (no tabs allowed in numpy code!)